aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -104,27 +104,95 @@ To define a JS initializer, add a JS module to the project named `{NAME}.lib.mod
 
 The module exports either or both of the following conventional functions:
 
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+<!-- UPDATE 8.0 Remove NOTE -->
+
+> [!NOTE]
+> JS initializers for Blazor Web Apps is an upcoming feature that will appear with the release of .NET 8 in mid-November.
+
+For Blazor Web Apps:
+
+* `beforeWebStart(options)`: Called before the Blazor Web App starts. For example, `beforeStart` is used to customize the loading process, logging level, and other options. Receives the Blazor options (`options`).
+* `afterWebStarted(blazor)`: Called after the Blazor Web App is ready to receive calls from JS. For example, `afterWebStarted` is used to initialize libraries by making JS interop calls and registering custom elements. The Blazor instance is passed to `afterWebStarted` as an argument (`blazor`).
+* `beforeServerStart(options, extensions)`: Called before each interactive Server runtime is started. Receives SignalR circuit start options (`options`) and any extensions (`extensions`) added during publishing.
+* `afterServerStarted(blazor)`: Called after each interactive Server runtime is started.
+* `beforeWebAssemblyStart(options, extensions)`: Called before each interactive WebAssembly runtime is started. Receives the Blazor options (`options`) and any extensions (`extensions`) added during publishing. For example, options can specify the use of a custom [boot resource loader](xref:blazor/fundamentals/startup#load-client-side-boot-resources).
```

Comment:
The WebAssembly runtime doesn't stop after it starts, so we can use phrasing that implies there's only ever one WebAssembly runtime:
```suggestion
* `beforeWebAssemblyStart(options, extensions)`: Called before the interactive WebAssembly runtime is started. Receives the Blazor options (`options`) and any extensions (`extensions`) added during publishing. For example, options can specify the use of a custom [boot resource loader](xref:blazor/fundamentals/startup#load-client-side-boot-resources).
```